src/rlbot_generated.rs

```diff
@@ -3874,7 +3874,8 @@ pub mod rlbot {
                     .get::<QuickChatSelection>(
                         QuickChat::VT_QUICKCHATSELECTION,
                         Some(QuickChatSelection::Information_IGotIt),
-                    ).unwrap()
+                    )
+                    .unwrap()
```

Comment:
`cargo fmt` keeps doing this for me, even though the format linting doesn't care either way. I just got tired of undoing it every time!

Reply:
I see your version of `rustfmt` actually formatted the other (ie better) way. I may have to upgrade or something.